akka-actor/src/main/mima-filters/2.5.26.backwards.excludes/28294-add-ExtensionId-apply-get-overloads.excludes

```diff
@@ -0,0 +1,12 @@
+# Incompatibilities against Akka < 2.5.17 where extra static bridge methods were generated that
```

Comment:
previously we added filters to 2.5.x.backwards.excludes, will this work when we release 2.5.27?

Reply:
IIRC `2.5.x.backwards.excludes` was only needed when we were making changes on master that would conflict with 'the latest 2.5.x' but hadn't released 2.6.0 yet. Nowadays adding them to `2.6.0.backwards.excludes` should be sufficient. In this particular case adding them to `2.5.16.backwards.excludes` might even be enough, since apparently mima doesn't get confused about akka versions >= 2.5.17? I'll try that.

Reply:
I think they can just go to 2.6.0. I had it already fixed but forgot to push, sorry.